### PR TITLE
MQTTwidget - support for the delivery of MQTT topic data realtime to your info-orbs! (uses latest DEV branch)

### PR DESCRIPTION
**Pull request by nutkracker**
A new PR that has been updated with the latest DEV that incorporates MQTTwidget - supports the delivery of MQTT topic data realtime to your info-orbs!

Updated readme included within the mqttwidget folder.

This uses the latest TTF fonts (which look great!) so you will need to adjust the X and Y of your label placements.  Those in the readme work.

Thank you.

Nutz

### CODE REVIEW

**Comment by flattermann** [comment]
Could you please rename the mqttWidget.cpp/.h to MqttWidget.cpp/.h? (We want to use UpperCaseFileNames for classes...)

And maybe you could remove the format changes in main.cpp to make the differences easier understandable?

Thanks!

**Review by lyricnz — APPROVED** [review]

File: firmware/src/widgets/mqttwidget/MQTTwidget-readme.txt [hunk 1]
```diff
@@ -0,0 +1,126 @@
+To enable MQTTWidget in your setup, do the following:
```
Comment (lyricnz) [inline firmware/src/widgets/mqttwidget/MQTTwidget-readme.txt:1]: This really could be markdown for better formatting, especially online.

File: info-orbs.code-workspace [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "[jsonc]": {
       "editor.defaultFormatter": "esbenp.prettier-vscode"
     },
-    "editor.formatOnSave": true
```
Comment (lyricnz) [inline info-orbs.code-workspace:17]: I'm going to reapply this setting when we merge the formatter PR https://github.com/brettdottech/info-orbs/pull/131

**Comment by brett-dot-tech** [comment]
Cloising this as it was merged via @flattermanns recfactor 👍

**Pull request closed without merging**